hanadb_exporter/main.py

```diff
@@ -97,10 +98,12 @@ def run():
         )
     except KeyError as err:
         raise KeyError('Configuration file {} is malformed: {} not found'.format(args.config, err))
-
+    logger.info('connection to hanadb sucessfully initialized')
```

Comment:
This entry is already logged in the `hdb_connector` (in shaptools). I think we can remove it

Reply:
ok thx